aether-producer/producer/__init__.py

```diff
@@ -302,6 +381,16 @@ class TopicManager(object):
         AND s.name = {schema_name}
         LIMIT 1; '''
 
+    # Count how many unique (controlled by kernel) messages should currently be in this topic
+    COUNT_STR = '''
+            SELECT
+                count(*)
```

Comment:
**count unique** would be `count(distinct e.id)`

Reply:
I think using distinct is slightly more expensive because by constraint we can't have duplicate entity IDs in the table, right?